meta-ov/recipes-apps/ovmenu-ng-skripts/files/upload-xcsoar.sh

```diff
@@ -2,15 +2,11 @@
 
 USB_PATH="/usb/usbstick/openvario/upload/xcsoar"
```

Comment:
BTW In newer versions the USB stick is mounted as "/usb" only - and not as "/usb/usbstick" like in previous versions

Reply:
this was an error #227 and corrected since.